lib/importer/src/functions.js

```diff
@@ -246,7 +245,7 @@ const parseNumberFromString = (s) => {
 }
 
 const importerRowHasHeader = (rowObj) => {
-    return rowObj.row.some( (element) => element.error )
+    return rowObj.row.some( (element) => element && element.error )
 }
```

Comment:
This looks like a bug fix for another issue? Which is fine, but just checking it is meant to be included in this PR.